apps/website/src/components/Placeholder.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { SvgInfoCircular } from '@itwin/itwinui-icons-react';
+
+export type Props = {
+  /** ending of the url for a specific storybook page */
+  componentPageName: string;
+};
+
+const { componentPageName } = Astro.props;
```

Comment:
you might want to take the rest props and add them to the root element
```
const { componentPageName, ...rest } = Astro.props;
---
<div {...rest}>
```

Reply:
Updated